_data/projectfooter.yaml

```diff
@@ -65,8 +65,8 @@ links:
       - page: English
         code: en
         url: https://quarkus.io/
-      - page: Português
-        code: pt
+      - page: Português (Brasileiro)
+        code: pt-br
```

Comment:
im surprised the footer still link to pt.quarkus.io after this change :)

Reply:
The url is set below this line, I believe this is used in the hreflang metadata only

Reply:
ah of course. gotcha.